cmd/agent/install_mac_os.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+# (C) Datadog, Inc. 2010-2016
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
```

Comment:
something I missed when I started the script: can you update the copyright and license with what we now use for Agent 6? See https://github.com/DataDog/datadog-agent/blob/master/cmd/agent/main.go#L1-L4 for example.